mods/d2k/chrome/ingame-observer.yaml

```
@@ -770,7 +778,14 @@ Container@OBSERVER_WIDGETS:
 									Align: Right
 									Shadow: True
 								Label@HARVESTERS:
-									X: 555
+									X: 525
```

```suggestion
									X: 555
```

You can't do that! It misalignes the whole UI